src/js/index.js

```diff
@@ -7,25 +7,31 @@ function getDiscussTopics() {
 	if (!section || !discussions) {
 		return;
 	}
-	fetch(`${Discussion.ORIGIN}/latest.json`).then(r => r.json()).then(r => {
-		const topics = r.topic_list.topics.slice(0, 2);
-		const allUsers = new Map(r.users.map(user => [user.id, user]));
-
-		if (topics && topics.length && allUsers.size) {
-			drawTopics(topics, allUsers);
-		}
-	}).then(_ => {
-		return fetch(`${Discussion.ORIGIN}/top.json`);
-	}).then(r => r.json()).then(r => {
+	let idList = [];
+	fetch(`${Discussion.ORIGIN}/top.json`).then(r => r.json()).then(r => {
```

Comment:
The order of top/latest was switched. We actually want to filter recent posts out of top, not the other way around.

Reply:
I realized that after I had pushed it :) How does this look?

Reply:
I fixed the top/latest priority issue and did a little bit of cleanup.